### PR TITLE
added leaflet map which displays cases/million of each state

### PR DESCRIPTION
**Pull request by goleary**
Added a map which shows cases & cases per million (styled based on cases per million) to the `/dashboard` route.
image

I didn't put much thought into the responsiveness of the map for now seeing as the charts above are also not responsive

### CODE REVIEW

File: _src/_assets/js/dashboard-map.js [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+      'https://api.mapbox.com/styles/v1/{id}/tiles/{z}/{x}/{y}?access_token={accessToken}',
+      {
+        attribution:
+          'Map data &copy; <a href="https://www.openstreetmap.org/">OpenStreetMap</a> contributors, <a href="https://creativecommons.org/licenses/by-sa/2.0/">CC-BY-SA</a>, Imagery © <a href="https://www.mapbox.com/">Mapbox</a>',
+        maxZoom: 18,
+        id: 'mapbox/light-v10',
+        tileSize: 512,
+        zoomOffset: -1,
+        accessToken:
+          'pk.eyJ1IjoiZ29sZWFyeSIsImEiOiJjaXJmNWh5YmgwMDd6ZzNuZXVsOHplYXRmIn0.41N9r7fWdPMGEz60wv5eZw',
```
Comment (schwartzadev) [inline _src/_assets/js/dashboard-map.js]: My guess is that this shouldn't be public?

Comment (goleary) [reply _src/_assets/js/dashboard-map.js]: it's a public token

Comment (schwartzadev) [reply _src/_assets/js/dashboard-map.js]: My bad then -- sorry!

**Comment by kissane** [comment]
Is this ready to push? It's going to the unlinked page, yes?

File: _src/dashboard.md [hunk 2]
```diff
@@ -9,10 +9,23 @@ nav: Dashboard
...
+   integrity="sha512-xwE/Az9zrjBIphAcBb3F6JVqxf46+CDLwfLMHloNu6KEQCAWi6HcDUbeOfBIptF7tcCzusKFjFw2yuvEpDL9wQ=="
+   crossorigin=""/>
+
+<link rel="stylesheet" href="/_assets/css/_dashboard.css" >
 <script src="/_assets/js/d3.js"></script>
 <script src="/_assets/js/britecharts.js"></script>
-<script src="/_assets/js/dashboard-charts.js"></script>
\ No newline at end of file
+
+ <script src="https://unpkg.com/leaflet@1.6.0/dist/leaflet.js"
```
Comment (olivierlacan) [inline _src/dashboard.md]: Can you fix your tag alignments here. They seem to be on different tab/space levels.

Comment (goleary) [reply _src/dashboard.md]: done

**Comment by goleary** [comment]
@kissane 
> Is this ready to merge? It's going to the unlinked page, yes?

not only is it an unlinked page, it's not the master branch so won't be published anyway.

**Pull request merged**